### ISSUE TITLE
plot_erpimage show_sortval doesnt work for categorical sortvals

### ISSUE THREAD

**Issue by behinger** [unfoldtoolbox/UnfoldMakie.jl#257]
if I do a string-categorical sortval I run into the error below.

MWE:
```julia
plot_erpimage(rand(5,10),sortvalues=string.(1:10),show_sortval=true)
```

```julia
ethodError: no method matching isnan(::String)


Closest candidates are:

isnan(!Matched::Missing)

@ Base missing.jl:101

isnan(!Matched::BigFloat)

@ Base mpfr.jl:982

isnan(!Matched::ForwardDiff.Dual)

@ ForwardDiff ~/.julia/packages/ForwardDiff/PcZ48/src/dual.jl:386

...


Stack trace
Here is what happened, the most recent locations are first:

_all @ reduce.jl:1288
all @ reducedim.jl:1023
ei_sortvalue(sortvalues::Observables.Observable{Vector{String}}, f::Makie.Figure, ax::Makie.Axis, hm::MakieCore.Heatmap{Tuple{Vector{Float64}, Vector{Float64}, Matrix{Float32}}}, config::UnfoldMakie.PlotConfig, sortval_xlabel::Observables.Observable{String}, sortplot_axis::@NamedTuple{}) @ plot_erpimage.jl:162
plot_erpimage!(f::Makie.Figure, times::Observables.Observable{StepRangeLen{Float64, Base.TwicePrecision{Float64}, Base.TwicePrecision{Float64}, Int64}}, data::Observables.Observable{Matrix{Float64}}; sortvalues::Vector{String}, sortindex::Observables.Observable{Nothing}, erpblur::Observables.Observable{Int64}, meanplot::Bool, show_sortval::Bool, sortval_xlabel::Observables.Observable{String}, meanplot_axis::@NamedTuple{}, sortplot_axis::@NamedTuple{}, kwargs::@Kwargs{}) @ plot_erpimage.jl:120
plot_erpimage!(::Makie.Figure, ::StepRangeLen{Float64, Base.TwicePrecision{Float64}, Base.TwicePrecision{Float64}, Int64}, ::Vararg{Any}; kwargs::@Kwargs{sortvalues::Vector{String}, show_sortval::Bool}) @ plot_erpimage.jl:56
UnfoldMakie @ plot_erpimage.jl:56
UnfoldMakie @ plot_erpimage.jl:49
```

**Comment by vladdez** [comment]
wait, but how i will sort, what picture do you expect? sequence number of x axis, and what is on y axis?

**Comment by behinger** [comment]
a categorical y-axis, like on a histogram maybe?

I had expected a step-function

**Comment by vladdez** [comment]
solved in #247
